tests/scripts/remote_monitoring_tests/rest_apis/test_update_performance_profile.py

```diff
@@ -149,6 +156,68 @@ def test_update_performance_profile_with_duplicate_data(cluster_type):
     print("delete performance profile = ", response.status_code)
 
 
+@pytest.mark.perf_profile
+def test_update_performance_profile_with_duplicate_slo_data(cluster_type):
```

Comment:
Please update the test docs as well to add new test cases and remove anything that is not applicable as per latest changes